modules/ROOT/pages/notifications/all-notifications.adoc

```diff
@@ -3,7 +3,7 @@
 [[listOfNnotifications]]
 = List of all server notification codes
 
-The following page provides an overview of all notifications in Neo4j, including the GQLSTATUS codes and the legacy Neo4j codes, along with some scenarios and their possible solutions.
+The following page provides an overview of all notifications in Neo4j, including the GQLSTATUS codes and Neo4j legacy codes, along with some scenarios and their possible solutions.
```

Comment:
Absolutely. My idea was to call all of them "Neo4j legacy codes" and then, depending on whether they were notifications or errors, legacy server error codes and legacy server notification codes, respectively.

Reply:
Actually, we can call all codes Neo4j error codes, and then we have the legacy Neo4j status codes (which comprise legacy notification codes and legacy error codes) and the GQLSTATUS codes. What do you think?

Reply:
Let's agree on one way of calling them and I'll update all the mentions in the manual.

Reply:
AFAIK, we use 'Neo4j status codes' to talk about old Neo4j notification and error codes not only in the manual but also across internal docs. I think it will confuse us and users if we start using 'Neo4j status codes' as an umbrella for both old and new (GQL-related) notification and error codes.

Let's keep 'Neo4j status codes' for the legacy notification and error codes.
At the moment, we use 'Neo4j notification codes', 'Neo4j error codes' (both are referred to the legacy codes) and GQLSTATUS notification codes and GQLSTATUS error codes to define newly added GQL-based status codes.

Talking about frameworks, we use 'Neo4j notification framework', 'Neo4j error framework' and  'GQL-status notification object', 'GQL-status error object' respectively. Right?

To be honest, I don't know what term to use if we want to cover both old and new status codes. Do we need such a term? I'm thinking only about descriptive options, for example: In Neo4j, both Neo4j status codes and GQL-based status codes are used. Neo4j status codes, which cover notifications and errors, are considered legacy codes.